packages/app/src/pages/ConfigurePracticeSession.vue

```diff
@@ -51,35 +47,45 @@ export default {
   data: function() {
     return {
       concepts: {
-        "Addition": {
+        Addition: {
           operator: Operator.Addition,
+          icon: "add"
         },
-        "Subtraction": {
+        Subtraction: {
           operator: Operator.Subtraction,
+          icon: "remove"
         },
-        "Multiplication": {
+        Multiplication: {
           operator: Operator.Multiplication,
+          icon: "close"
         }
       }
-    }
+    };
   },
   methods: {
     getToPracticeSession($event) {
       this.$router.push("/");
     }
   },
   components: {
-    ConceptPickerItem: ConceptPickerItem,
+    ConceptPickerButton: ConceptPickerButton,
     ModeSelector: ModeSelector,
     DifficultyPicker: DifficultyPicker
-  },
+  }
 };
 </script>
 
 <style scoped>
 .body-style {
   padding-top: 30px;
-  padding-left: 5em;
-  padding-right: 5em;
+  padding-left: 3em;
+  padding-right: 3em;
+}
+h6 {
+  font-family: Varela Round;
```

Comment:
To use this font, we have to import it

Reply:
https://fonts.google.com/specimen/Varela+Round?query=varela

Reply:
![image](https://user-images.githubusercontent.com/48028572/85689065-f3168600-b6ef-11ea-9803-ef1451f73248.png)

**This is how I am doing it is it okay ?**

Reply:
Use this guide: https://quasar.dev/style/typography#Add-custom-fonts